### DISCUSSION TITLE
APK Installer v0.1.4

### DISCUSSION THREAD

**Discussion by wherewhere** [Paving-Base/APK-Installer-Classic#26] [Announcements]
###### 更新日志
1. 更新版本号至 v0.1.4
2. 修复了一些 Bug
3. 修改了一些 UI
4. 同步 WinUI 版功能
5. 将平台迁移到 .net4.8
6. 现在可以拖拽安装应用了
###### 下载链接
[![Microsoft Store](https://img.shields.io/badge/download-%e4%b8%8b%e8%bd%bd-magenta.svg?label=Microsoft%20Store&logo=Microsoft&style=for-the-badge&color=11a2f8)](https://www.microsoft.com/store/apps/9N3HJLJP8V15 "Microsoft Store")
###### 屏幕截图
![安装](https://raw.githubusercontent.com/Paving-Base/APK-Installer-Classic/screenshots/Screenshots/v0.1.4/Snipaste_2022-07-20_22-31-31.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/Paving-Base/APK-Installer-Classic/releases/tag/v0.1.4'>APK Installer v0.1.4</a>.</em>

**Comment by Dospy-Honor** [comment]
好耶！是where大叔

**Reply by wherewhere** [reply]
喵喵喵？

**Comment by Dospy-Honor** [comment]
是我啊，Honor（，葵酱对象来着



---Original---
From: ***@***.***&gt;
Date: Thu, Jul 28, 2022 12:39 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [Paving-Base/APK-Installer-Classic] APK Installer v0.1.4 (Discussion #26)




 
喵喵喵？

**Reply by wherewhere** [reply]
喵喵喵？

**Comment by Yue-cn** [comment]
喵喵喵？

**Reply by wherewhere** [reply]
喵喵喵？
